tests/x11/x3270_ssl.pm

```diff
@@ -70,7 +70,14 @@ sub run {
     # Launch x3270
     # Add noverifycert option if x3270 is or greater than v3.6ga
     my $noverifycert = ($current_ver < 3.6 ? '' : '-noverifycert');
-    script_run "x3270 -trace $noverifycert -tracefile $tracelog_file L:localhost:8443";
+
+    # Export SYSTEMD_PAGER to let the journalctl exits automatically
+    assert_script_run('export SYSTEMD_PAGER=""');
```

Comment:
Ben, do we have to this? I agree adding this line if no harmless

Reply:
I think it is not necessary for the solution. Remove it is also fine for me.

Reply:
Done.